### PR TITLE
Adding SLSA index and tutorial

### PR DESCRIPTION
**Pull request by sheesh**

### CODE REVIEW

**Comment by sheesh** [comment]
Giving @jspeed-meyers & @tracymiranda an opportunity to review if desired. Otherwise, ready for a DevEd team review.

**Comment by jspeed-meyers** [comment]
Reviewed via Google Doc and provided comments there. Nice work, @sheesh!

File: content/open-source/slsa/what-is-slsa.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+The stricter requirements for Level 2 help provide more protection against software tampering and enable greater levels of trust that the provenance data is accurately represented. 
+
+### Level 3 
+**The source and build platforms meet specific standards to guarantee the auditability of the source and the integrity of the provenance respectively.** 
+
+This level aims to increase trust and harden infrastructure through a variety of requirements designed to meet specific threats. The requirements are as follows:
+
+* **[Build as code](https://slsa.dev/spec/v0.1/requirements#build-as-code)**: You must be able to verifiably derive the build definition and configuration from text file definitions stored in a version control system.
```
Comment (ltagliaferri) [inline content/open-source/slsa/what-is-slsa.md:51]: I really like the use of bullets here, I think you can do something similar in the other sections to strengthen this and make it easier for readers to scan the document

File: content/open-source/slsa/what-is-slsa.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Generating provenance compliant with Level 3 requirements can be challenging, but this information is increasingly important to help end users verify the integrity of the software before implementing it. Recently, SLSA released the free SLSA 3 Container Generator for GitHub Actions that helps ease the process by allowing you to build automated provenance generation into your container workflows. To learn more about how the SLSA 3 Container Generator works, visit the [announcement blog post](https://slsa.dev/blog/2023/02/slsa-github-workflows-container-ga). You can also check out SLSA's guide on [Reaching Level 3](https://slsa.dev/get-started#reaching-slsa-level-3). 
+
+### Level 4 
+This level represents the highest level of trust possible in the SLSA framework and provides the strongest possible assurance that the software has not been tampered with. It is one of the most difficult levels to achieve, and may not be necessary (or even possible) for all software projects.
```
Comment (ltagliaferri) [inline content/open-source/slsa/what-is-slsa.md:60]: Is there a missing bold statement here?

File: content/open-source/slsa/what-is-slsa.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+A _reproducible_ build is one in which the build steps will always reproduce “bit-for-bit” identical output. If software cannot meet this requirement, a justification must be provided. Though the build service may not verify reproducibility, consumers should be able to check its reproducibility and can reject the build if they cannot reproduce the results.
+
+## SLSA Tools and Practices
+
+SLSA is a relatively new framework, with supporting tools and practices still actively evolving. Some of the requirements listed in the levels above can be met using popular build and version control systems. More specific requirements may require additional tooling, and SLSA hosts some supporting tools in its [Github repository](https://github.com/slsa-framework). As mentioned in the description of Level 3, [SLSA released a tool for automating provenance generation](https://slsa.dev/blog/2023/02/slsa-github-workflows-container-ga), one of the hardest requirements, with Github Actions in February 2023.
```
Comment (ltagliaferri) [inline content/open-source/slsa/what-is-slsa.md:69]:
```suggestion
SLSA is a relatively new framework, with supporting tools and practices still actively evolving. Some of the requirements listed in the levels above can be met using popular build and version control systems. More specific requirements may require additional tooling, and SLSA hosts some supporting tools in its [GitHub repository](https://github.com/slsa-framework). As mentioned in the description of Level 3, [SLSA released a tool for automating provenance generation](https://slsa.dev/blog/2023/02/slsa-github-workflows-container-ga), one of the hardest requirements, with GitHub Actions in February 2023.  
```

File: content/open-source/slsa/what-is-slsa.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+## SLSA Tools and Practices
+
+SLSA is a relatively new framework, with supporting tools and practices still actively evolving. Some of the requirements listed in the levels above can be met using popular build and version control systems. More specific requirements may require additional tooling, and SLSA hosts some supporting tools in its [Github repository](https://github.com/slsa-framework). As mentioned in the description of Level 3, [SLSA released a tool for automating provenance generation](https://slsa.dev/blog/2023/02/slsa-github-workflows-container-ga), one of the hardest requirements, with Github Actions in February 2023.  
+
+To verify the SLSA provenance of a piece of software, you can use the [`slsa-verifier` tool](https://github.com/slsa-framework/slsa-verifier), which verifies provenances generated by the `slsa-generator` tool or Google Cloud Build. Other tools, like Chainguard’s [Enforce](https://www.chainguard.dev/chainguard-enforce) or Sigstore’s open source [Policy Controller](https://docs.sigstore.dev/policy-controller/overview/) allow you to create policies around SLSA requirements in your Kubernetes cluster.
```
Comment (ltagliaferri) [inline content/open-source/slsa/what-is-slsa.md:71]: "provenances" sounds weird, I would try to rephrase to keep that word singular

File: content/open-source/slsa/what-is-slsa.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+To verify the SLSA provenance of a piece of software, you can use the [`slsa-verifier` tool](https://github.com/slsa-framework/slsa-verifier), which verifies provenances generated by the `slsa-generator` tool or Google Cloud Build. Other tools, like Chainguard’s [Enforce](https://www.chainguard.dev/chainguard-enforce) or Sigstore’s open source [Policy Controller](https://docs.sigstore.dev/policy-controller/overview/) allow you to create policies around SLSA requirements in your Kubernetes cluster. 
+
+Developers are also encouraged to include the corresponding SLSA level badge in their readme once their codebase meet the level’s requirements.
```
Comment (ltagliaferri) [inline content/open-source/slsa/what-is-slsa.md:73]:
```suggestion
Developers are also encouraged to include the corresponding SLSA level badge in their README once their codebase meets the level’s requirements. 
```

File: content/open-source/slsa/what-is-slsa.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+While SLSA provides a strong framework for verifying the authenticity and integrity of software, it is important to note that it does not protect against every type of supply chain attack. For example, SLSA requirements cannot prevent attacks enabled through vulnerable code, vulnerabile build platforms, or collusion between high level actors. Still, SLSA offers a powerful framework for defending against common supply chain threats, and will likely emerge as a standard component of modern software as tooling and community adoption evolves. 
+
+To learn more about SLSA, you can visit the [SLSA website](https://slsa.dev/), read an [in-depth overview of SLSA requirements](https://slsa.dev/spec/v0.1/requirements), or explore the [SLSA repository on Github](https://github.com/slsa-framework).
```
Comment (ltagliaferri) [inline content/open-source/slsa/what-is-slsa.md:81]:
```suggestion
To learn more about SLSA, you can visit the [SLSA website](https://slsa.dev/), read an [in-depth overview of SLSA requirements](https://slsa.dev/spec/v0.1/requirements), or explore the [SLSA repository on GitHub](https://github.com/slsa-framework).  
```

**Review by ltagliaferri — COMMENTED** [review]
Thank you so much for writing the first piece on SLSA! I left some comments, I think I am not totally crisp on everything myself so feel free to update me. I know you have some other feedback you need to incorporate so ignore what is no longer useful here.

File: content/open-source/slsa/_index.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+---
+title: "SLSA"
```
[the remark this answers is not in the document]
Comment (sheesh) [reply content/open-source/slsa/_index.md:2]: added - thx!

File: content/open-source/slsa/what-is-slsa.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+[Released by Google’s Open Source Security Team](https://security.googleblog.com/2021/06/introducing-slsa-end-to-end-framework.html) in 2001, SLSA was created to help software creators understand where and how they can harden their supply chain security practices, and help software consumers better evaluate whether a software product or component is safe to use. SLSA was also designed around the creation of verifiable metadata, so that software consumers can set automated policies to prevent the deployment of code that does not meet their preferred SLSA level. 
+
+Today, SLSA is a vendor-neutral project supported by the [Open Source Security Foundation](https://openssf.org/) and is actively evolving its standards and tooling with industry input. In this guide, you will learn about SLSA levels and security requirements, emerging tools that can help you meet these requirements, and where you can learn more about the framework.
```
[the remark this answers is not in the document]
Comment (sheesh) [reply content/open-source/slsa/what-is-slsa.md:22]: SLSA is a framework, but they have some tools in their repo to support the framework. Good flag though! I adjusted the language to make clear that SLSA is a framework but that the org also develops some tools to support the framework.

File: content/open-source/slsa/what-is-slsa.md (hunk 2 above)
Comment (sheesh) [reply content/open-source/slsa/what-is-slsa.md:60]: Ah, thanks -- added it back!

File: content/open-source/slsa/what-is-slsa.md (hunk 3 above)
Comment (sheesh) [reply content/open-source/slsa/what-is-slsa.md:69]: thanks for catching!

File: content/open-source/slsa/what-is-slsa.md (hunk 4 above)
Comment (sheesh) [reply content/open-source/slsa/what-is-slsa.md:71]: done!

**Comment by sheesh** [comment]
Either the deploy function is taking longer than usual, or it's not working. In any case, I need to check to see if the placement SLSA badge images works. 

Also, @ltagliaferri it looks like your edits for the vexctl tutorial are somehow associated. Let me know if you want me to go ahead and resolve them here.

**Review by ltagliaferri — COMMENTED** [review]
Check the menu config

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**